packages/create-strapi-app/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "create-strapi-app",
+  "version": "3.0.0-beta.9",
+  "description": "Generate a new Strapi application.",
+  "license": "MIT",
+  "homepage": "http://strapi.io",
+  "keywords": [
+    "create-strapi-app",
+    "create",
+    "new",
+    "generate",
+    "strapi"
+  ],
+  "main": "./index.js",
+  "bin": {
+    "create-strapi-app": "./index.js"
+  },
+  "files": [
+    "index.js"
+  ],
+  "dependencies": {
+    "commander": "^2.20.0",
+    "strapi-generate-new": "3.0.0-beta.9"
+  },
+  "scripts": {
+    "test": "echo \"no tests yet\""
+  },
+  "author": {
+    "email": "hi@strapi.io",
+    "name": "Strapi team",
+    "url": "http://strapi.io"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/strapi/strapi.git"
+  },
+  "bugs": {
+    "url": "https://github.com/strapi/strapi/issues"
+  },
+  "engines": {
+    "node": ">= 8"
```

Comment:
Yeah we should be forcing node 10 as node 8 is EOL

Reply:
It is intended. Right now there are still a lot of people using node v8? If someone on an older version installs the package on node v8 we will tell him to go to v10 It will be a much better experience https://github.com/strapi/strapi/pull/3497/files#diff-8bc1fb9d45d579968c2dbe9fd70e6ca9R8

Reply:
Ah okay you it's set to node 8 but it will check if the user has node 10 and if not give them that instead of some deep and confusing error message :P Gotcha